next.clayui.com/content/docs/components/modal.mdx

```diff
@@ -29,6 +32,147 @@ You may want to compose your content and customize with your use cases or add yo
 
 <Modal />
 
+### useModal hook
+
+The `useModal` [hook](https://reactjs.org/docs/hooks-intro.html) is required to create a Modal component in Clay, it is responsible for communicating with ClayModal and controlling states to make animations, this is necessary because Modal needs animation when opening and closing so we need to delay calls of the `onClose` so you can safely control Modal's visibility or do other things.
+
+`useModal` returns a signature with `observer` and `onClose`:
+
+- **observer** - Observer is the communication channel that connects ClayModal with `useModal`, you can call it a tunnel.
```

Comment:
Not sure the "you can call it a tunnel." part is helpful, it might add to the confusion.

Reply:
Hmm yeah, I will remove it.